src/Apps/Collect2/Components/SeoProductsForCollection.tsx

```diff
@@ -0,0 +1,132 @@
+import { SeoProductsForCollection_artworks } from "__generated__/SeoProductsForCollection_artworks.graphql"
+import { Product } from "Components/v2/Seo/Product"
+import React from "react"
+import { createFragmentContainer, graphql } from "react-relay"
+
+export interface SeoProductsForCollectionProps {
+  artworks: SeoProductsForCollection_artworks
+  collectionDescription: string
+  collectionURL: string
+  collectionName: string
+}
+
+// dont have anything about currency conversion in Artsy right now.
+// Just use the fixed ratio to show the approximated price.
+export function simpleConversionToUSD(price: number, currencyCode: string) {
+  if (currencyCode === "USD") {
+    return price
+  } else if (currencyCode === "GBP") {
+    return price * 1.28
+  } else if (currencyCode === "EUR") return price * 1.1
+}
+
+export const getMaxMinPrice = priceableArtworks => {
+  const priceRange = [null, null]
```

Comment:
#minor I have slight preference having an object instead of array here to make the following lines more readable (`priceRange.min` instead of `priceRange[0]`)

Also it is possible to set the initial values to `-Number.MAX_VALUE`/`Number.MAX_VALUE` to avoid the checks like `!priceRange[1] || ...`

```js
const priceRange = { min: Number.MAX_VALUE, max: - Number.MAX_VALUE } 
```

However this is just a stylistic suggestion so if you are more comfortable with the current implementation it is perfectly fine.